packages/devtools_app/lib/src/screens/deep_link_validation/deep_links_controller.dart

```diff
@@ -424,16 +428,28 @@ class DeepLinksController extends DisposableController {
     }
     if (appLinkSettings.error != null) {
       pagePhase.value = PagePhase.analyzeErrorPage;
+      ga.select(
+        gac.deeplink,
+        gac.AnalyzeFlutterProject.flutterAppLinkLoadingError.name,
```

Comment:
should this be differentiated from the error above? or is it fine to use the same event?